test/data/base_units.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sbml xmlns="http://www.sbml.org/sbml/level3/version2/core" level="3" version="2">
+    <model id="case00935" name="case00935">
```

Comment:
if this is a testcase from SBML, it should be taken from SBML. If not, we need to write down copyright.

Reply:
It definitely doesn't look like https://github.com/sbmlteam/sbml-test-suite/blob/42cbda7a271053eccb90034e42d5c95a6cb049ce/cases/semantic/00935/00935-sbml-l3v2.xml